lib/tokens/token.js

```diff
@@ -60,7 +60,6 @@ module.exports = function (log, crypto, P, hkdf, Bundle, error) {
           Token.deriveTokenKeys(TokenType, bytes)
             .then(
               function (keys) {
-                details.createdAt = Date.now()
```

Comment:
It's not obvious to me why we were forcing that; with it removed, it just falls back to calling Date.now() in the `TokenType` constructor which calls Date.now() by default, right?